src/Features/Core/Portable/Completion/SharedSyntaxContextsWithSpeculativeModel.cs

```diff
@@ -43,8 +43,8 @@ public Task<SyntaxContext> GetSyntaxContextAsync(Document document, Cancellation
             // Extract a local function to avoid creating a closure for code path of cache hit.
             static AsyncLazy<SyntaxContext> GetLazySyntaxContextWithSpeculativeModel(Document document, SharedSyntaxContextsWithSpeculativeModel self)
             {
-                return self._cache.GetOrAdd(document, d => AsyncLazy.Create(cancellationToken
-                    => CompletionHelper.CreateSyntaxContextWithExistingSpeculativeModelAsync(d, self._position, cancellationToken), cacheResult: true));
+                return self._cache.GetOrAdd(document, d => AsyncLazy.Create(
+                    c => CompletionHelper.CreateSyntaxContextWithExistingSpeculativeModelAsync(d, self._position, c)));
```

Comment:
📝 `CancellationToken` parameters should be named `cancellationToken`. A corollary to this is `c` is not a `CancellationToken`.

Reply:
I'd be fine with an analyzer that requires this. However, 'c' for cancellation token is a super common pattern in our codebase used widely:-)